content/user/new_user_topics/no_such_file.md

```diff
@@ -34,13 +36,41 @@ this file would be `includes/templates/responsive_classic/templates/tpl_product_
 But what if the file still doesn't exist, even if you now know the real 
 name of the file? 
 
-Template files (those people write about with `YOURTEMPLATE`) are all
-altered copies of the files in one directory above. 
+Template files (the ones with `YOURTEMPLATE` - the [template name](/user/first_steps/basic_terms/#yourtemplate) in their path) are all
+altered copies of the default file.
+
+### How do you find the default file? 
+
+a) For files under `/includes/templates`, the default file is `includes/templates/template_default/FOLDER/FILENAME`. 
+
+b) For all other files, the default file is one level above the template folder. 
+
+If the templated file being referenced doesn't exist in your cart, you can 
+create it by copying the original from the default file. 
+
+[Learn more about default files](/user/first_steps/overrides/#default-files) 
+
+### Installing a file
+
+Unless you are creating these files directly on your server, 
+you need to copy these files to your server.  Use your 
+[FTP tool](/user/first_steps/useful_tools/#ftp-tools) to connect to your server, then go to your [webroot](/user/first_steps/how_do_i_install#what-is-my-webroot),
+then to the folder where the file is supposed to exist. 
+
+Example: 
+- Your webroot is `/home/johndoe/public_html`
+- Your template is `custom`
+- The file you are changing is `includes/languages/custom/english.php`
 
-So `includes/modules/responsive_sheffield_blue/additional_images.php` is a templated file based on `includes/modules/additional_images.php`.
+Steps: 
+- Using FTP, connect to `YOURSITE.com`.  
+- Change directories so you get to the webroot.  
+- Change directories from there so that you get to the folder for the file you are changing.  At this point, you will be in `/home/johndoe/public_html/includes/languages/custom`
+- Transfer the file.
 
-Likewise, `includes/languages/responsive_classic/english.php` is a templated modification of `includes/languages/english.php`.
+Here's what it looks like for my store.  Click the image to make it larger if needed.  As you can see, my template is `bootstrap` and my webroot is `/home/thatsoft/public_html`.
```

Comment:
Did you want this to be personalized this heavily? "my store", "my template", "thatsoft"

Reply:
The trouble is I wanted it to match the image.   But I can certainly eliminate the "my" usage.

Reply:
I'm not bothered by it. Just wanted to be sure you meant it.